src/file.cr

```diff
@@ -2,22 +2,20 @@ require "crystal/system/file"
 
 class File < IO::FileDescriptor
   # The file/directory separator character. `'/'` in Unix, `'\\'` in Windows.
-  SEPARATOR = {% if flag?(:windows) %}
-    '\\'
-  {% else %}
-    '/'
-  {% end %}
+  SEPARATOR = '/'
 
   # The file/directory separator string. `"/"` in Unix, `"\\"` in Windows.
-  SEPARATOR_STRING = {% if flag?(:windows) %}
-    "\\"
-  {% else %}
-    "/"
-  {% end %}
+  SEPARATOR_STRING = "/"
 
   # :nodoc:
   DEFAULT_CREATE_PERMISSIONS = File::Permissions.new(0o644)
 
+  {% if flag?(:win32) %}
+    DEVNULL = "NUL"
```

Comment:
Is this const used anywhere?

Reply:
So `:nodoc:`?

Reply:
...no, why would an application not need this?

Reply:
Then it should be documented with a proper description.

Reply:
Oh yeah, good point. Thanks.